numpy/ma/core.py

```diff
@@ -2755,6 +2755,62 @@ class MaskedArray(ndarray):
         in any order (either C-, Fortran-contiguous, or even discontiguous),
         unless a copy is required, in which case it will be C-contiguous.
 
+     Examples
+    --------
+
+    You can initialize MaskedArray with ``mask`` attribute being array
+    of boolean values with the same shape as ``data`` attribute:
+    >>> np.ma.MaskedArray(np.ones([1, 2]), mask=[[False, False]])
+    masked_array(data=[[1.0, 1.0]],
+                 mask=[[False, False]],
+                 fill_value=1e+20)
+    >>> np.ma.MaskedArray(np.ones([1, 2]), mask=[[False, True]])
+    masked_array(data=[[1.0, --]],
+                 mask=[[False, True]],
+                 fill_value=1e+20)
+
+    You can also initialize MaskedArray with ``mask`` attribute being
+    scalar boolean value:
```

Comment:
It would be nice if the text here briefly describes what initializing `mask` with a scalar boolean value does:
```suggestion
    Alternatively, the ``mask`` can be initialized to a homogeneous
    boolean array with same shape as ``data`` by passing 
    in scalar boolean values:
```